zerver/views/report.py

```diff
@@ -36,5 +35,4 @@ def get_attr(csp_report_attr: str) -> str:
         get_attr("status-code"),
         get_attr("script-sample"),
     )
-
```

Comment:
I'd prefer you not remove this newline; it seems intended to improve readability by separating parts of the function.